src/providers/vllm-provider.ts

```
@@ -302,40 +315,40 @@
   async selectModelForTask(task: 'japanese' | 'code' | 'general' | 'fast'): Promise<string> {
     const availableModels = await this.getAvailableModels();
-    
+
     switch (task) {
       case 'japanese':
         // Prefer Japanese-specific models
-        const japaneseModels = availableModels.filter(m => 
-          m.includes('japanese') || m.includes('jp')
+        const japaneseModels = availableModels.filter(
+          (m) => m.includes('japanese') || m.includes('jp'),
         );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)